src/test/regress/sql/alter_table_set_access_method_view.sql

```diff
@@ -0,0 +1,14 @@
+CREATE SCHEMA view_acceess_method_test;
+SET search_path TO view_acceess_method_test;
```

Comment:
Can we move this test to file `alter_table_set_access_method.sql`?

Reply:
Fixed you as you suggested